addons/im_livechat/models/rating_rating.py

```diff
@@ -22,6 +22,14 @@ def _compute_res_name(self):
     def action_open_rated_object(self):
         action = super(Rating, self).action_open_rated_object()
         if self.res_model == 'discuss.channel':
+            if self.env[self.res_model].browse(self.res_id).is_member:
+                ctx = self.env.context.copy()
+                ctx.update({'active_id': self.res_id})
+                return {
+                    'type': 'ir.actions.client',
+                    'tag': 'mail.action_discuss',
+                    'context': ctx,
+                }
```

Comment:
Missing `name: _("Discuss")` (otherwise it keeps showing "Livechat")

Reply:
Let's postpone this in another PR, as there's this small problem in other contexts (e.g. open a chat window in discuss app)